linter/lintfuncs.go

```diff
@@ -12,11 +12,13 @@ const (
 	ThisUpdateLimitSubscriber = "96h" // 4 days
 	ProducedAtLimitCA = "8760h" // 365 days
 	ThisUpdateLimitCA = "8760h" // 365 days
+	NextUpdateLimitSubscriber = "240h" // 10 days
```

Comment:
just curious, is there a glossary of what these parameters mean?

Reply:
As we discussed in our meeting today, if I do end up formalizing the `subscriber`, `subordinate CA`, `non-issued` certificate types, I'll include it in `CONTRIBUTING.md`